### PR TITLE
Fix/e mail template url

### LINKED ISSUE #49: Fix verify e-mail url on HTML temaplte

### PR DESCRIPTION
**Pull request by VerasThiago**
Closes #49

### CODE REVIEW

File: shared/clients/email/helper.go [hunk 1]
```diff
@@ -15,10 +15,10 @@ import (
 
 const (
 	VERIFY_EMAIL_TEMPLATE_PATH   = "../shared/templates/verify_email.html"
-	QR_CODE_TEMPLATE_PATH        = "../shared/templates/email.html"
+	NEW_TICKET_TEMPLATE_PATH     = "../shared/templates/new_ticket.html"
 	RESET_PASSWORD_TEMPLATE_PATH = "../shared/templates/forgot_password.html"
 	INVITE_TITLE                 = "Despedida Veras - Convite"
-	TICKETS_TILE                 = "Despedida Veras - Ingressos"
+	TICKET_TILE                  = "Despedida Veras - Ingresso"
```
Comment (diogoftp) [inline shared/clients/email/helper.go:21]: I think there is a typo here. Is it supposed to be `title`?

File: shared/templates/new_ticket.html [hunk 2]
```diff
@@ -114,25 +114,24 @@
...
-                        <p style="color: #f2f2f2; font-family: sans-serif; font-size: 18px; font-weight: 400; line-height: 22px; letter-spacing: 0px; text-align: center; margin: 0; margin-bottom: 15px; padding-bottom: 16px;">Para conferir seus ingressos, clique no botão abaixo:</p>
+                        <p style="color: #f2f2f2; font-family: sans-serif; font-size: 18px; font-weight: 400; line-height: 22px; letter-spacing: 0px; text-align: center; margin: 0; margin-bottom: 15px; padding-bottom: 16px;">Você acabou de receber um ingresso novinho no nome de:</p>
+                        <p style="color: #f2f2f2; font-family: sans-serif; font-size: 18px; font-weight: 400; line-height: 22px; letter-spacing: 0px; text-align: center; margin: 0; margin-bottom: 15px; padding-bottom: 16px;">{{.TicketUserName}}</p>
+                        <p style="color: #f2f2f2; font-family: sans-serif; font-size: 18px; font-weight: 400; line-height: 22px; letter-spacing: 0px; text-align: center; margin: 0; margin-bottom: 15px; padding-bottom: 16px;">Vai lá dar uma olhadinha nos seus ingresso, boa festa!</p>
```
Comment (diogoftp) [inline shared/templates/new_ticket.html:119]: Typo `nos seus ingresso`

**Review by diogoftp — APPROVED** [review]

File: shared/templates/forgot_password.html [hunk 3]
```diff
@@ -130,8 +130,6 @@
                             </tr>
                           </tbody>
                         </table>
-                        <p class="more-info" style="color: #f2f2f2; font-family: sans-serif; font-weight: 400; line-height: 22px; letter-spacing: 0px; text-align: center; margin: 0; margin-bottom: 15px; padding-bottom: 16px; font-size: 14px;">Se tiver problemas, copie e cole o link abaixo em outra janela do seu navegador:</p>
-                        <p class="url-link" style="font-family: sans-serif; font-weight: 400; line-height: 22px; letter-spacing: 0px; margin: 0; margin-bottom: 15px; padding-bottom: 16px; text-align: center; font-size: 14px; color: #763DD3; text-decoration: none!important;;"><a class="url-link" href="{{.Url}}">{{.Url}}</a></p>
```
Comment (limarodrigoo) [inline shared/templates/forgot_password.html:134]: more-info and url-link won't be used anymore, what about remove them from the tag style?

**Review by limarodrigoo — APPROVED** [review]

File: shared/templates/forgot_password.html (hunk 3 above)
Comment (VerasThiago) [reply shared/templates/forgot_password.html:134]: Good point

Comment (VerasThiago) [reply shared/templates/forgot_password.html:134]: `more-info` don't exist on this file, only `url-link`

**Review by limarodrigoo — APPROVED** [review]

**Pull request merged**